dev/box_setup.sh

```diff
@@ -12,7 +12,9 @@ function dexec() {
 dexec surface ./manage.py migrate
 
 # create local admin (ignore failure as user might already exist)
-dexec surface ./manage.py createsuperuser --noinput --username admin --email admin@localhost && dexec surface ./manage.py changepassword admin
+
+#dexec surface ./manage.py createsuperuser --noinput --username admin --email admin@localhost
+dexec surface ./manage.py changepassword admin
```

Comment:
```suggestion

dexec surface ./manage.py createsuperuser --noinput --username admin --email admin@localhost
```

As discussed then, reverting this change to the original docker execution.